tools/attempt_protocols.js

```diff
@@ -15,10 +16,11 @@ if (argv._.length >= 1) {
 
 const gamedig = new GameDig(options)
 
-const protocols = ['valve', 'gamespy1', 'gamespy2', 'gamespy3', 'goldsrc', 'minecraft', 'quake1', 'quake2', 'quake3', 'unreal2', 'valve']
+let protocolList = []
+Object.keys(protocols).forEach((key) => protocolList.push(key))
 
 const run = async () => {
-  for (const protocol of protocols) {
+  for (const protocol of protocolList) {
```

Comment:
I think its the same thing to do `for (const protocol of Object.keys(protocols))`, could you check?

Reply:
Nice! Indeed it is! Thank you! ✨